scene/gui/graph_edit.cpp

```diff
@@ -40,13 +40,13 @@
 #include "editor/editor_scale.h"
 #endif
 
-#define ZOOM_SCALE 1.2
+const float ZOOM_SCALE = 1.2f;
 
-#define MIN_ZOOM (((1 / ZOOM_SCALE) / ZOOM_SCALE) / ZOOM_SCALE)
-#define MAX_ZOOM (1 * ZOOM_SCALE * ZOOM_SCALE * ZOOM_SCALE)
+const float MIN_ZOOM = powf(1.0f / ZOOM_SCALE, 8.0f);
+const float MAX_ZOOM = powf(ZOOM_SCALE, 3.0f);
 
-#define MINIMAP_OFFSET 12
-#define MINIMAP_PADDING 5
+const int MINIMAP_OFFSET = 12;
+const int MINIMAP_PADDING = 5;
```

Comment:
Can't comment on this, a core dev who decides code style stuff might have to give their thoughts.

Reply:
`const` (or better, `constexpr`) should be preferred over symbolic defines in modern C++.

Reply:
I would have done `constexpr` but it seems it only appears in thirdparty code from what I can see, should I use `constexpr`?

Reply:
There are some plans to change some macros to `constexpr` (see #45263 for example), so it's ok to use them.